bundles/org.openhab.binding.asuswrt/src/main/resources/OH-INF/i18n/asuswrt.properties

```diff
@@ -0,0 +1,118 @@
+# binding
```

Comment:
This file (and the source where the i18n gets its strings from) have sefverall occurences of the same string, but are not consistent. I'm not a native English speaker, so not sure what is correct, but i thikn it should atleast be consistent.
IP-address vs IP address
MAC-address vs MAC address
DNBS-name DNS Name